<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMainWindow</class>
 <widget class="QMainWindow" name="UserMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="picAgoDisplay">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>430</width>
      <height>340</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>盖章前照片</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="picAfterDisplay_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>430</width>
      <height>340</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>盖章后照片</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二维码信息：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QRcodelineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{border: 1.3px solid grey; border-radius:1px; background-color:rgba(255,255,255,0%);}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>360</width>
      <height>290</height>
     </rect>
    </property>
    <property name="title">
     <string>解析结果</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>盖章种类：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>盖章数量：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>剩余次数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用 章 人：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="proposerLineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{border: 1.3px solid grey; border-radius:1px; background-color:rgba(255,255,255,0%);}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sealTypeLineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{border: 1.3px solid grey; border-radius:1px; background-color:rgba(255,255,255,0%);}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="totalNumberLineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{border: 1.3px solid grey; border-radius:1px; background-color:rgba(255,255,255,0%);}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="remainingNumberLineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{border: 1.3px solid grey; border-radius:1px; background-color:rgba(255,255,255,0%);}</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>360</width>
      <height>270</height>
     </rect>
    </property>
    <property name="title">
     <string>系统状态</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>扫 描 仪：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>相    机：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>盖章系统：</string>
     </property>
    </widget>
    <widget class="QLabel" name="QRcodeScannerLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>45</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>通信正常</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="cameraLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>115</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>通信正常</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="slaveSystemLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>185</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>通信正常</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>430</width>
      <height>270</height>
     </rect>
    </property>
    <property name="title">
     <string>操作面板</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>盖 章</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>391</width>
       <height>25</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>29</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>盖章进度显示</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>申请重盖</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>400</y>
      <width>430</width>
      <height>270</height>
     </rect>
    </property>
    <property name="title">
     <string>再次申请</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>45</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>申 请 人：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>95</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>申请时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>145</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>申请原因：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmBtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>195</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确 认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>195</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>重 置</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="proposerAgainLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>45</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="timeLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>95</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{border: 1.3px solid grey; border-radius:1px; background-color:rgba(255,255,255,0%);}</string>
     </property>
    </widget>
    <widget class="QComboBox" name="reasonComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>145</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1340</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
